examples/lustre/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# ch-test-scope: full
+# ch-test-arch-exclude: aarch64
+# No lustre RPMS for aarch64
+FROM centos7
+
+# install development tools
+RUN yum install -y epel-release
+
+# more dependencies
+RUN yum -y install libyaml-devel \
+                   wget \
+                   gcc
+
+ENV LUSTRE_URL=https://downloads.whamcloud.com/public/lustre/latest-release/el7/client/RPMS/x86_64
+
+RUN wget ${LUSTRE_URL}/lustre-client-2.12.3-1.el7.x86_64.rpm && \
+    wget ${LUSTRE_URL}/kmod-lustre-client-2.12.3-1.el7.x86_64.rpm
+
+RUN yum localinstall -y *.rpm
+
+WORKDIR /lustre
```

Comment:
This instruction will have no effect in Charliecloud, since it's the last one in the Dockerfile. It looks like you are using it for the directory creation side effect? If so, I'd recommend `RUN mkdir ...`.